src/ConfigEditor/migrating-from-datasource-http-settings.md

```diff
@@ -0,0 +1,88 @@
+# Migrating from DataSourceHttpSettings component
+
+## Overview
+
+As `DataSourceHttpSettings` is considered deprecated, `@grafana/experimental` package has a few components that will help you to migrate away from it.
+
+According to the [datasource configuration best practices](https://docs.google.com/document/d/11XMaYHSMSra8AoN2-yVW-5QPbjEXgkAFbTIF6N84cGE/edit) the data source configuration page should consist of multiple sections, so there is no single component that can just replace `DataSourceHttpSettings` component. Instead, there are 3 components that will take care of different parts of the `DataSourceHttpSettings` and they can be placed in different places inside the datasource config editor. This is how mapping old -> new looks like:
+
+<img src="./docs-img/migrate-from-datasource-http-settings.png" width="600">
+
+New components are:
+
+- [ConnnectionSettings](./Connection)
```

Comment:
Typo on ConnectionSettings

Reply:
Fixed in a3fcb9bd1b9aaa410b4d4f830e16f53db89da98a.